compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,263 @@
+# onecc-docker
+
+`onecc-docker` supports `one-cmd`, which is currently only supported by ubuntu 18.04, in more OS.
+
+
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So people in other environments can't use our tools unless they upgrade the OS (or install Ubuntu OS).
+
+Therefore, we developed `onecc` that runs using a docker so that users can use `one-cmds` more easily. This is `onecc-docker.`
+
+> Currently, `onecc-docker` is only supported in Linux environments.
```

Comment:
Why is ther a `>` ?

Reply:
When I usually organize documents, I used BlockQuote for further explanation. I checked other documents, and it doesn't seem common. I'll fix it!